fennel/datasets/aggregate.py

```diff
@@ -8,13 +8,15 @@
 
 
 class AggregateType(BaseModel):
-    window: Union[Continuous, Hopping, Session, Tumbling]
+    window: Optional[Union[Continuous, Hopping, Session, Tumbling]]
```

Comment:
why is this Optional ?

Reply:
Because now we can choose one window directly from groupby("a", window=Something).agg(count=Count())